SSI.php

```diff
@@ -936,7 +940,7 @@ function ssi_randomMember($random_type = '', $output_method = 'echo')
 	}
 
 	// Get the lowest ID we're interested in.
-	$member_id = mt_rand(1, $modSettings['latestMember']);
+	$member_id = random_int(1, $modSettings['latestMember']);
 
```

Comment:
According to this: https://stackoverflow.com/questions/44228718/php-rand-vs-random-int
It seems that random_int is slower.  For stuff like this, I don't think we need to use a more secure function.

Reply:
Well i used this code:
```
$loop  = 10000;

$trash = mt_rand(0,10000);
var_dump($trash);
$trash = random_int(0,10000);
var_dump($trash);

$start = microtime(true);
for($i = 0; $i < $loop; $i++)
    $trash = mt_rand(0,10000);

$time_new2 = microtime(true) - $start;

$start = microtime(true);
for($i = 0; $i < $loop; $i++)
    $trash = random_int(0,10000);

$time_new = microtime(true) - $start;




var_dump($time_new);
var_dump($time_new2);
```
the diff doesn't look so big:
D:\Programme\xampp2\htdocs\test11.php:10:int 8922

D:\Programme\xampp2\htdocs\test11.php:12:int 8651

D:\Programme\xampp2\htdocs\test11.php:29:float 0.013932943344116

D:\Programme\xampp2\htdocs\test11.php:30:float 0.011417150497437

Maybe the result is different on linux oder server systems...

Feel free to define which are not needed.